addons/pos_restaurant/static/src/overrides/components/receipt_screen/receipt_screen.js

```diff
@@ -27,7 +27,11 @@ patch(ReceiptScreen.prototype, {
         this.pos.showScreen("ProductScreen");
     },
     isContinueSplitting() {
-        if (this.pos.config.module_pos_restaurant && this.pos.selectedTable) {
+        if (
+            this.pos.config.module_pos_restaurant &&
+            !this.pos.selectedTable &&
+            !this.currentOrder.uiState.originalSplittedOrderFinalized
```

Comment:
```suggestion
            !this.currentOrder.originalSplittedOrder.finalized
```

Why doesn't this work?

Reply:
Oops, thank you for pointing that out. Could you please check again?